hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/handlers/InventoryHandlers.java

```diff
@@ -61,6 +61,17 @@
      */
 
     @GET
+    @Path("/export")
+    @Produces(APPLICATION_JSON)
+    public Response exportInventory() {
```

Comment:
Same here, pagination should be needed.
If we don't want pagination we can treated it as multipart streaming, and pagination is only in the InventoryService.
I don't know, but this is hit a OOME in the preliminar tests of QE.